### ISSUE TITLE
Allow users to 'publish' model output, e.g. create searchable database of results by user

### ISSUE THREAD

**Issue by billspat** [krishnanlab/geneplexus_app#136]
Users should be a able to share outputs and their input geneses with other users of the system, and for other features.    Will use the same application database as the users IDs.  

dependent on #134

**Comment by billspat** [comment]
features and issues deferred for rewrite --closing here.
